Protectli no shell and no ipxe timeout

miczyg1 opened the pull request.

miczyg1 committed:
payloads/external/edk2/Kconfig: Do not include Shell by default

Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

miczyg1 committed:
configs: Update edk2 revision so it can build without Shell

Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

miczyg1 committed:
configs/config.protectli_vp2420: Bump to v1.2.1-rc5

Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

miczyg1 committed:
configs/config.protectli_vp32xx: Bump to v0.9.0-rc8

Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

macpijan reviewed (dismissed):
lgtm provided that we resolve https://github.com/Dasharo/edk2/pull/208/files first

miczyg1 committed:
configs/config.protectli_vp2420: Bump to v1.2.1-rc6

Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

miczyg1 committed:
payloads/external/iPXE/Kconfig: Use proper default for IPXE_SERIAL_CONSOLE

Enabling IPXE_SERIAL_CONSOLE with EDK2 payload or SeaBIOS payload with sercon
would cause the	output to be doubled. SeaBIOS sercon already handles the VGA
INT 10H	redirection to serial, so iPXE by printing to VGA and to serial, would
result in SeaBIOS and iPXE printing to serial port. In case of EDK2, iPXE uses
ConOut to print the output. In a case where EDK2 enables serial console and
adds it	to ConOut, iPXE	would print the output two times on ConOut and serial
port explicitly. By disabling the IPXE_SERIAL_CONSOLE we let the payload print
the output on the serial console.

Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

miczyg1 committed:
configs/config.protectli_vp2430: Disable fan curves and efivar option backed

VP2430 does not come with any fans so disable fan curves.

Disable UEFI variable option backed as it is not used by Dasharo.

Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

miczyg1 committed:
configs/config.protectli_vp2430: Fix broken option

Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

miczyg1 committed:
configs/config.protectli_vp2420: Bump to v1.2.1-rc7

Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

miczyg1 committed:
configs/config.protectli_vp32xx: Bump to v0.9.0-rc9

Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>
